xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -416,7 +416,8 @@
     "items": { "type": "string",
       "description": "Requesting the genreid, artistid and/or albumartistid field will result in increased response times",
       "enum": [ "title", "artist", "albumartist", "genre", "year",
-                "rating", "votes", "userrating", "album", "track", "duration", "comment",
+                "rating", "votes", "userrating", "album", "track", "duration", "comment", 
+				"mood", "contributors", "composer", "conductor", "orchestra", "lyricist",
```

Comment:
To keep backward compat with non named params, new fields needs to always be added at the end.

+ Some indentation problem